piano-hero/core/src/main/java/com/novoda/pianohero/Sequence.java

```diff
@@ -0,0 +1,42 @@
+package com.novoda.pianohero;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+public final class Sequence {
+
+    private final List<Set<Note>> notes;
+
+    private Sequence(List<Set<Note>> notes) {
```

Comment:
this is implicitly ordered - you have to use the Builder to create this and you don't get access to the list.